lib/easyoptions.c

```diff
@@ -354,6 +355,6 @@ struct curl_easyoption Curl_easyopts[] = {
  */
 int Curl_easyopts_check(void)
 {
-  return ((CURLOPT_LASTENTRY%10000) != (310 + 1));
+  return (CURLOPT_LASTENTRY != (311 + 1));
```

Comment:
This came when regenerating the file via `optiontable.pl`; if the `%10000` is important, I can add it back (and/or include a commit to add it in the template in `optiontable.pl`)

Reply:
It needs the `%10000` there in one way or another, because the last number will be the enum number used on the line above `+1`, and since the number above might be in a numerical range above 10000, doing the logic without the remainder operator could evaluate wrong.

Reply:
Ah, that makes sense now after looking at https://github.com/curl/curl/blob/e5d5e288dd62bf64bdf336152c2391191bea66b7/include/curl/curl.h#L1031-L1041

I'll push a commit to fix that, hopefully later tonight (in a few hours)